short-read-mngs/experimental.wdl

```diff
@@ -247,6 +247,10 @@ task RunGlueJob {
       exit 0 
     elif [[ $JOBSTATE == "FAILED" ]]; then 
       exit 1
+    elif [[ $JOBSTATE == "ERROR" ]]; then 
+      exit 1
+    elif [[ $JOBSTATE == "TIMEOUT" ]]; then 
+      exit 1
     else 
       sleep 60
```

Comment:
I think STOPPED should be added as a case at least in which we exit. I guess it would exit on the timeout. That would be an edge case in which the job gets manually stopped via CLI or console

Reply:
Hm, good point I wasn't sure if STOPPED would be an error state or a success state. Or if it would eventually become one. Do you know if there is any chance that a STOPPED state would eventually lead to SUCCESSFUL?